manifests/0000_20_kube-apiserver-operator_01_config.crd.yaml

```diff
@@ -263,7 +52,7 @@ spec:
             managementState:
               description: managementState indicates whether and how the operator
                 should manage the component
-              pattern: ^(Managed|Force)$
+              pattern: ^(Managed|Unmanaged|Force|Removed)$
```

Comment:
this is wrong. The patch is not applied. Maybe you have to rename or change it? Compare my unmerged PR switching to controller-tools.

Reply:
Hm, good catch. It seems like the name is right, because update codegen is trying to apply it with yq:
```
$ make update-codegen
Using existing controller-gen from "_output/tools/src/sigs.k8s.io/controller-tools/controller-gen"
Using existing yq from "_output/tools/bin/yq"
'_output/tools/src/sigs.k8s.io/controller-tools/controller-gen' \
		schemapatch:manifests="./manifests" \
		paths="./vendor/github.com/openshift/api/operator/v1" \
		output:dir="./manifests"
cp -n ./manifests/0000_20_kube-apiserver-operator_01_config.crd.yaml-merge-patch './manifests/' || true  # FIXME: centos
cp: ./manifests/0000_20_kube-apiserver-operator_01_config.crd.yaml-merge-patch and ./manifests/0000_20_kube-apiserver-operator_01_config.crd.yaml-merge-patch are identical (not copied).
_output/tools/bin/yq m -i './manifests/0000_20_kube-apiserver-operator_01_config.crd.yaml' './manifests/0000_20_kube-apiserver-operator_01_config.crd.yaml-merge-patch'
```
but it's silently failing. maybe the yaml in the patch is malformed, looking more

Reply:
think I got it, the command needs `-x`: `-x, --overwrite     update the yaml file by overwriting existing values`

```
$ _output/tools/bin/yq m -i -x './manifests/0000_20_kube-apiserver-operator_01_config.crd.yaml' './manifests/0000_20_kube-apiserver-operator_01_config.crd.yaml-merge-patch'
```

produces the right change. I'll open a pull to library-go to update

Reply:
@sttts this should be resolved now, bumped to include the fix